exchanges/deribit/deribit_websocket_eps.go

```diff
@@ -0,0 +1,2139 @@
+		Currency string `json:"currency,omitempty"`
+		Count    int64  `json:"count,omitempty"`
+		Offset   int64  `json:"offset,omitempty"`
+	}{
+		Currency: currency,
+		Count:    count,
+		Offset:   offset,
+	}
+	var resp WithdrawalsData
+	return &resp, d.SendWSRequest(getWithdrawals, input, &resp, true)
+}
+
+// WSSubmitTransferToSubAccount submits a request to transfer a currency to a subaccount
+func (d *Deribit) WSSubmitTransferToSubAccount(currency string, amount float64, destinationID int64) (*TransferData, error) {
+	if currency == "" {
+		return nil, fmt.Errorf("%w \"%s\"", errInvalidCurrency, currency)
+	}
+	if amount <= 0 {
+		return nil, errInvalidAmount
+	}
+	if destinationID <= 0 {
+		return nil, errors.New("invalid destination address")
+	}
+	input := &struct {
+		Currency    string  `json:"currency"`
+		Destination int64   `json:"destination"`
+		Amount      float64 `json:"amount"`
+	}{
+		Currency:    currency,
+		Destination: destinationID,
+		Amount:      amount,
+	}
+	var resp TransferData
+	return &resp, d.SendWSRequest(submitTransferToSubaccount, input, &resp, true)
+}
+
+// WSSubmitTransferToUser submits a request to transfer a currency to another user through the websocket connection.
+func (d *Deribit) WSSubmitTransferToUser(currency, tfa, destinationAddress string, amount float64) (*TransferData, error) {
+	if currency == "" {
+		return nil, fmt.Errorf("%w \"%s\"", errInvalidCurrency, currency)
+	}
+	if amount <= 0 {
+		return nil, errInvalidAmount
+	}
+	if destinationAddress == "" {
+		return nil, errors.New("invalid destination address")
+	}
+	input := &struct {
+		Currency                    string  `json:"currency"`
+		TwoFactorAuthenticationCode string  `json:"tfa,omitempty"`
+		DestinationID               string  `json:"destination"`
+		Amount                      float64 `json:"amount"`
+	}{
+		Currency:                    currency,
+		TwoFactorAuthenticationCode: tfa,
+		DestinationID:               destinationAddress,
+		Amount:                      amount,
+	}
+	var resp TransferData
+	return &resp, d.SendWSRequest(submitTransferToUser, input, &resp, true)
+}
+
+// ----------------------------------------------------------------------------
+
+// WSSubmitWithdraw submits a withdrawal request to the exchange for the requested currency through the websocket connection.
+func (d *Deribit) WSSubmitWithdraw(currency, address, priority string, amount float64) (*WithdrawData, error) {
+	if currency == "" {
+		return nil, fmt.Errorf("%w \"%s\"", errInvalidCurrency, currency)
+	}
+	if amount <= 0 {
+		return nil, errInvalidAmount
+	}
+	if address == "" {
+		return nil, errInvalidCryptoAddress
+	}
+	if priority != "" && priority != "insane" && priority != "extreme_high" && priority != "very_high" && priority != "high" && priority != "mid" && priority != "low" && priority != "very_low" {
+		return nil, errors.New("unsupported priority '%s', only insane ,extreme_high ,very_high ,high ,mid ,low ,and very_low")
+	}
```

Comment:
This was requested to be done prior, please go over all requested nits and make sure they are complete. [Here](https://github.com/thrasher-corp/gocryptotrader/pull/1082/files#r1020980474)